invokeai/app/invocations/tiles.py

```diff
@@ -56,6 +65,86 @@ def invoke(self, context: InvocationContext) -> CalculateImageTilesOutput:
         return CalculateImageTilesOutput(tiles=tiles)
 
 
+@invocation(
+    "calculate_image_tiles_Even_Split",
+    title="Calculate Image Tiles Even Split",
+    tags=["tiles"],
+    category="tiles",
+    version="1.0.0",
+)
+class CalculateImageTilesEvenSplitInvocation(BaseInvocation):
+    """Calculate the coordinates and overlaps of tiles that cover a target image shape."""
+
+    image_width: int = InputField(ge=1, default=1024, description="The image width, in pixels, to calculate tiles for.")
+    image_height: int = InputField(
+        ge=1, default=1024, description="The image height, in pixels, to calculate tiles for."
+    )
+    num_tiles_x: int = InputField(
+        default=2,
+        ge=1,
+        description="Number of tiles to divide image into on the x axis",
+    )
+    num_tiles_y: int = InputField(
+        default=2,
+        ge=1,
+        description="Number of tiles to divide image into on the y axis",
+    )
+    overlap: float = InputField(
+        default=0.25,
+        ge=0,
+        lt=1,
+        description="Overlap amount of tile size (0-1)",
+    )
+
+    def invoke(self, context: InvocationContext) -> CalculateImageTilesOutput:
+        tiles = calc_tiles_even_split(
+            image_height=self.image_height,
+            image_width=self.image_width,
+            num_tiles_x=self.num_tiles_x,
+            num_tiles_y=self.num_tiles_y,
+            overlap=self.overlap,
+        )
+        return CalculateImageTilesOutput(tiles=tiles)
+
+
+@invocation(
+    "calculate_image_tiles_min_overlap",
+    title="Calculate Image Tiles Minimum Overlap",
+    tags=["tiles"],
+    category="tiles",
+    version="1.0.0",
+)
+class CalculateImageTilesMinimumOverlapInvocation(BaseInvocation):
+    """Calculate the coordinates and overlaps of tiles that cover a target image shape."""
+
+    image_width: int = InputField(ge=1, default=1024, description="The image width, in pixels, to calculate tiles for.")
+    image_height: int = InputField(
+        ge=1, default=1024, description="The image height, in pixels, to calculate tiles for."
+    )
+    tile_width: int = InputField(ge=1, default=576, description="The tile width, in pixels.")
+    tile_height: int = InputField(ge=1, default=576, description="The tile height, in pixels.")
+    min_overlap: int = InputField(
+        default=128,
+        ge=0,
+        description="minimum tile overlap size (must be a multiple of 8)",
+    )
```

Comment:
Originally this had a "multiple_of=8,"  restriction that I missed moving over.  I will re-add this restriction and update the description.

Reply:
Isn't this unnecessarily limiting? E.g. if you wanted to use this node for image-space tiling.

Reply:
Discussed with @JPPhoto and the text has changed and the restrictions removed

Reply:
It looks like the implementation (`calc_tiles_even_split(...)`) still applies multiple-of-8 rounding / assertions. Whichever way we go, we should make sure that the interface assertions/documentation match the implementation.

Reply:
The `calc_tiles_even_split(...)` has to do this as there is no way for a user to enter actual tile sizes or overlap sizes.  Without this, it would regularly output coordinates that are not on the 8-pixel boundaries.